ga4gh/serverconfig.py

```diff
@@ -26,6 +26,12 @@ class BaseConfig(object):
     SIMULATED_BACKEND_VARIANT_DENSITY = 0.5
     SIMULATED_BACKEND_NUM_VARIANT_SETS = 1
 
+    # Options for OIDC
+    # OIDC_PROVIDER = "https://accounts.google.com"
+    # OIDC_CLIENT_ID = "XXXX"
+    # OIDC_CLIENT_SECRET = "XXXX"
+    SERVER_NAME = "localhost:8001"
```

Comment:
And also flask really doesn't want you to know the port it's running on, and url_for doesn't work if you're running on anything but port 80, unless that port is specified in SERVER_NAME. Anyway, I think I've worked something out. I'm adding the port manually (which means a new parameter to frontend.configure).